src/fabio_rodrigues_ex/ex01/letter_counts.py

```diff
@@ -0,0 +1,16 @@
+def letter_freq(txt):
+    """ Returns a dictionary = {(key : value)}, from the input string, with key = letters and
+    value = the number of occurrences of the letter in the given string.
+    letters = [i for i in text]
+    counter = [text.count(a) for a in text]
+    return dict(zip(letters, counter))
+    """
+    return dict([(i, txt.count(i)) for i in txt])
```

Comment:
Case is ignored in this function, i.e, `h` is treated the same with `H` so you may need to use `lower()` function.